etc/kube/pachyderm.json

```diff
@@ -348,24 +346,7 @@
   },
   "status": {
     "replicas": 0
-  }
```

Comment:
Change due to pachctl being updated in a prior commit but us not running `make kube-cluster-assets`.